Remotion/Validation/Core/RuleCollectors/AddingPropertyValidationRuleCollector1.cs

```diff
@@ -1,4 +1,20 @@
-﻿using System;
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
```

Comment:
I added here also the missing file header since it was in the same namespace `Remotion.Validation.RuleCollectors`. @MichaelKetting Is that okay?

Reply:
Is it just a random one and there's others or was this the last file? I do would prefer to keep those in separate PRs (don't need to review those :) so if you find typos or missing cleanup in unrelated files, you can just make a ticket and a PR and fix it that way. it keeps the original PR clean and still allows for cleanup.

Reply:
yes I think it was the last one.  I will revert this change and make a ticket